fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/api/FixedDepositAccountsApiResource.java

```diff
@@ -268,8 +271,11 @@ public String update(@PathParam("accountId") final Long accountId, final String
     @Path("{accountId}")
     @Consumes({ MediaType.APPLICATION_JSON })
     @Produces({ MediaType.APPLICATION_JSON })
-    public String handleCommands(@PathParam("accountId") final Long accountId, @QueryParam("command") final String commandParam,
-            @Context final UriInfo uriInfo, final String apiRequestBodyAsJson) {
+    @ApiOperation(value = "Approve fixed deposit application | Undo approval fixed deposit application | Reject fixed deposit application | Withdraw fixed deposit application | Activate a fixed deposit account | Close a fixed deposit account | Premature Close a fixed deposit account | Calculate Premature amount on Fixed deposit account | Calculate Interest on Fixed Deposit Account | Post Interest on Fixed Deposit Account", httpMethod = "POST", notes = "Approve fixed deposit application:\n" + "Approves fixed deposit application so long as its in 'Submitted and pending approval' state.\n" + "Undo approval fixed deposit application:\n" + "Will move 'approved' fixed deposit application back to 'Submitted and pending approval' state.\n" + "Reject fixed deposit application:\n" + "Rejects fixed deposit application so long as its in 'Submitted and pending approval' state.\n" + "Withdraw fixed deposit application:\n" + "Used when an applicant withdraws from the fixed deposit application. It must be in 'Submitted and pending approval' state.\n" + "Close a fixed deposit account:\n" + "Results in a Matured fixed deposit account being converted into a 'closed' fixed deposit account.\n" + "Premature Close a fixed deposit account:\n" + "Results in an Active fixed deposit account being converted into a 'Premature Closed' fixed deposit account with options to withdraw prematured amount. (premature amount is calculated using interest rate chart applicable along with penal interest if any.)\n" + "Calculate Premature amount on Fixed deposit account:\n" + "Calculate premature amount on fixed deposit account till premature close date. Premature amount is calculated based on interest chart and penal interest applicable.\n" + "Calculate Interest on Fixed Deposit Account:\n" + "Calculates interest earned on a fixed deposit account based on todays date. It does not attempt to post or credit the interest on the account. That is responsibility of the Post Interest API that will likely be called by overnight process.\n" + "Post Interest on Fixed Deposit Account:\n" + "Calculates and Posts interest earned on a fixed deposit account based on today's date and whether an interest posting or crediting event is due.\n" + "Showing request/response for Calculate Interest on Fixed Deposit Account")
```

Comment:
![screenshot from 2017-12-19 01 44 51](https://user-images.githubusercontent.com/11756370/34126172-ac8d7e70-e45e-11e7-92bc-693d73c3b4cc.png)

Indentation of notes section is not correct
please add by adding extra new line characters
like: "\n\n" + "\n" + ""